reana_server/reana_admin/cli.py

```diff
@@ -419,7 +424,13 @@ def _format_statuses(type_, statuses):
 
         if email:
             send_email(email, f"{hostname} system status report", status_report_body)
-            click.echo(f"Status report successfully sent by email to {email}.")
+            if REANA_NOTIFICATIONS_ENABLED:
```

Comment:
If we decide to throw an exception instead of just writing to the logs, then we do not need this check